shared/output/gapic/templates/garbage/lib/so/much/trash/resource_names/paths.rb

```diff
@@ -31,6 +31,23 @@ module Trash
       module ResourceNames
         # Path helper methods for the ResourceNames API.
         module Paths
+          ##
+          # Create a fully-qualified ComplexPatternLogParent resource string.
+          #
+          # The resource will be in the following format:
+          #
+          # `returns/{return_order}~{return_item}`
+          #
+          # @param return_order [String]
+          # @param return_item [String]
+          #
+          # @return [::String]
+          def complex_pattern_log_parent_path return_order:, return_item:
+            raise ::ArgumentError, "return_order cannot contain /" if return_order.to_s.include? "/"
```

Comment:
It looks like the logic that is deciding which arguments can or cannot contain `/` needs to be updated. Since this PR is about updating tests, then we probably just need to file an issue to fix this separately.

Reply:
good catch! I've created https://github.com/googleapis/gapic-generator-ruby/issues/465 and will make a pass on it when I have a cycle